### ISSUE TITLE
swipe down to dismiss

### ISSUE THREAD

**Issue by DJafari** [alexvasilkov/GestureViews#58]
how can implement swipe down image in viewpager to dismiss activity ?

unfortunately if i enable zoom gesture, SwipeBack libraries failed to work
i test this :
https://github.com/liuguangqiang/SwipeBack

**Comment by alashow** [comment]
Just enable it:
```java
image.getController().getSettings()
            .setExitEnabled(true);
```

**Comment by DJafari** [comment]
@alashow thanks, but i have a activity with viewpager, i dont use animation from list/grid view to viewpager, i have standalone activity with viewpager, i want when swipe down an image, activity finished, can give me a sample ?

**Comment by alexvasilkov** [comment]
This is not really an intended use case, exit gesture is tightly bound to position animator for a smoother animation.
But this is how you can do it in Ex1 (Image in ViewPager) in sample app:

Add next to the end of the onCreate() method:

```Java
 // Creating transition animator
 final ViewsTransitionAnimator<Integer> animator = new ViewsTransitionBuilder<Integer>()
         .intoViewPager(viewPager, new SimpleViewsTracker() {
             @Override
             public View getViewForPosition(int position) {
                 RecyclePagerAdapter.ViewHolder holder = adapter.getViewHolder(position);
                 return holder == null ? null : PaintingsPagerAdapter.getImage(holder);
             }
         })
         .build();

 animator.setFromListener(new ViewsCoordinator.OnRequestViewListener<Integer>() {
     @Override
     public void onRequestView(@NonNull Integer id) {
         animator.setFromNone(id);
     }
 });

 // Listening for exit animation to start
 animator.addPositionUpdateListener(new ViewPositionAnimator.PositionUpdateListener() {
     @Override
     public void onPositionUpdate(float position, boolean isLeaving) {
         if (!isFinishing() && isLeaving
                 && animator.getToView().getPositionAnimator().isAnimating()) {
             animator.getToView().getPositionAnimator().stopAnimation();
             finish();
         }
     }
 });

 // Entering without animation, to allow exit gesture
 animator.enter(0, false);
```

**Comment by alexvasilkov** [comment]
Note, that `animator.setFromNone(id);` method is not available in last version (2.2.0), so please either wait for 2.2.1 or use latest library code directly.

**Comment by DJafari** [comment]
@alexvasilkov working goood, thanks

i will waiting to next release
